### DISCUSSION TITLE
Treeview looks terrible? A1111 1.8

### DISCUSSION THREAD

**Discussion by suonnon** [lobehub/sd-webui-lobe-theme#543] [General]
Since the update 1.8 has been released, the new view for extranetworks/checkpoints (tree view) becomes not compatible with the UI/UX of Lobetheme. I prefer the old tab browser for folders instead of the tree view, or just leave it as a dropdown menu box is best I guess?

**Comment by grandfromages** [comment]
I agree!!! on a 4K monitor, I can't make the treeview section wider so I always have to guess which category I'm in - it's very frustrating! The tab/folder buttons on the top were perfect as is (perhaps let us make their font smaller so I can see more categories buttons at once? I have many). I strongly considered switching to something else over this, but Lobe is the best option. Please just make it a switch or something on the settings. (I'm super surprised it was added without the option to switch back.)
Thank you!
